code/core/src/phone/java/com/adobe/marketing/mobile/services/ui/AEPMessage.java

```diff
@@ -169,99 +184,97 @@ public void show(final boolean withMessagingDelegateControl) {
                 () -> {
                     final AEPMessage message = this;
 
+                    // create the webview if needed
```

Comment:
The description seems to state that the web view will be created when AEPMessage is constructed -but looks like this is still being done onShow(). Looks like I am missing something here... 😅

Reply:
you are correct... i had originally planned to create the webview when the aepmessage is constructed but moved it back here after the conversation we had with praveen. he mentioned the webview should be created using the executor.

Reply:
fixed the title and description to better reflect the current behavior